### ISSUE TITLE
Mouse clicks rarely work... delay shipping?

### ISSUE THREAD

**Issue by TylerLeonhardt** [PowerShell/GraphicalTools#95]
Because of this issue https://github.com/migueldeicaza/gui.cs/issues/386 that @tig opened, mouse clicks rarely work _anywhere_.

* I can't click the list view
* I can't click the filter box
* And the status bar isn't clickable (separate issue but adds to this)

This is very unfortunate because although the experience is good for keyboard users, it's un-usable for mouse users.

I wanted to use my mouse... but I also got tripped up using the keyboard:

Because I was sitting in the filter box, and wanted to get to the list view... I, without thinking, hit `ENTER` since I do this in basically any GUI search box (google.com, outlook search, github search, etc). Because `ENTER` is our keybinding to `Accept`, the grid view went away.

Even I was bamboozled about this and I work on the thing 😅

At the very least, we need to add that `TAB` switches between the list view and search box in the status bar.

With that said, I'm not really sure if we wanna ship yet. The mouse support is ideal.

**Comment by SteveL-MSFT** [comment]
Looks like there was a PR that was merged to improve mouse perf.  If we can get a new gui.cs release with that change, we can verify if this is still an issue.

**Comment by tig** [comment]
>At the very least, we need to add that TAB switches between the list view and search box in the status bar.

@TylerLeonhardt not sure what you mean by this. Tab currently switches between the filter edit box and the list. Are you suggesting it should also go to status bar? I don't think it should. Everything on the status bar is actionable with a keystroke. This is also different from how most GUI apps with status bars work (tab never takes you there). 

FWIW, mouse is working fine for me in `ConEmu` and `cmd.exe` in Windows. Of course there's no mouse support in `Windows Terminal` yet. I built with `Termina.gui` `0.81.0`. 

What platform are you on.

**Comment by TylerLeonhardt** [comment]
@tig I'm on macOS - iTerm2 is my terminal.

**Comment by TylerLeonhardt** [comment]
Someone suggested we release what we have as a prerelease on the Gallery allowing folks to opt-in to the better keyboard experience but degraded mouse experience. Seems like a good idea.

**Comment by tig** [comment]
I'm a fan of doing that.

We're rapidly converging on a `0.90` release of `Terminal.gui` where mouse performance will be much better. I plan on diving into `ocgv` this week to update it to `0.90` and ensure we didn't break anything. The goal is to have `0.90` released this week or next.

**Comment by TylerLeonhardt** [comment]
That'd be perfect

**Comment by tig** [comment]
Head's up, `0.90` is imminent. 

I've got a PR ready for `GraphicalTools` for when it's out.

https://github.com/migueldeicaza/gui.cs/issues/792

**Comment by TylerLeonhardt** [comment]
awesome!

**Comment by TylerLeonhardt** [comment]
@tig how are we looking? :)

**Comment by tig** [comment]
Close. I just pushed a test of `0.90`'s nuget package and am currently tweaking my PR for `ocgv`. Hopefully today!

**Comment by TylerLeonhardt** [comment]
@tig how's it going? Need any help?

**Comment by tig** [comment]
> @tig how's it going? Need any help?

Appreciate the prod. I think we know what it is and how to fix. I am hoping to find time today.

What I need to do:

1) Update GraphicalTools to use the pre-0.90 branch of gui.cs (`<PackageReference Include="Terminal.Gui" Version="0.89.4" />`) - Some of the breaking changes impacted GraphicalTools. Done. But I can't remember what machine I did it on and it doesn't look like I pushed to my github.
2) Debug `Shutdown` issue - Fix is (apparently) to explicitly call ` Application.Shutdown();` - See https://github.com/migueldeicaza/gui.cs/pull/880
